src/crimson/osd/osd.cc

```diff
@@ -676,7 +678,8 @@ void OSD::print(std::ostream& out) const
   out << "{osd." << superblock.whoami << " "
       << superblock.osd_fsid << " [" << superblock.oldest_map
       << "," << superblock.newest_map << "] " << pg_shard_manager.get_num_pgs()
-      << " pgs}";
+      << " pgs " << "tlb" << superblock.cluster_osdmap_trim_lower_bound
```

Comment:
Move tlb to just after the map ranges (just before the number of pgs).  Also, "tlb" -> "tlb: "